frontend/sozisel-app/src/components/PresenterSession/EventsTimline/LiveEventDetails/TextSection.tsx

```diff
@@ -0,0 +1,16 @@
+import "./LiveEventDetails.scss";
+import React, { ReactElement } from "react";
+
+interface TextSectionProps {
+  children?: ReactElement;
```

Comment:
If you want to use `children` you could simply use `React.FC` as it has built-in children in its props

Reply:
I changed it, but I had to bump the version of `"eslint-plugin-react"` to `"^7.25.1"` because I got props type error